indico/htdocs/js/indico/modules/eventdisplay.js

```diff
@@ -23,6 +23,47 @@ $(document).ready(function() {
         });
     });
 
+    var threeRowsHeight = $('.height-control').height();
+    $('.participant-list-wrapper').toggleClass('collapsible collapsed',
+        $('.participant-list').height() > threeRowsHeight);
+    var initialHeight = $('.participant-list-wrapper').height();
+
+    $('.participant-list-wrapper > .trigger, .participant-list-wrapper > .gradient-layer').on('click', function() {
+        var toggler = $('.participant-list-wrapper > .trigger');
+        var participantList = toggler.siblings('.participant-list');
+        var wrapper = participantList.parent();
+        if (wrapper.hasClass('collapsed')) {
+            var newHeight = participantList.height();
+            participantList.height(initialHeight);
+            wrapper.find('.gradient-layer').fadeOut();
+            wrapper.toggleClass('collapsed');
```

Comment:
Why not `removeClass`? toggling when you already know the state you have/want seems a bit weird